main/gameData.c

```diff
@@ -4,10 +4,7 @@
 
```

Comment:
Please move this file into the `platformer` folder

Reply:
Fixed